mkosi/__init__.py

```diff
@@ -1897,6 +1897,17 @@ def build_microcode_initrd(context: Context) -> list[Path]:
     return [microcode]
 
 
+def finalize_kernel_modules_include(context: Context, *, include: Sequence[str], host: bool) -> list[str]:
```

Comment:
The order of the includes shouldn't be important, since it only goes into a regex, right? Then we should maybe make this a set to deduplicate.